installers-conda/resources/spyder-menu.json

```diff
@@ -12,6 +12,7 @@
             "platforms": {
                 "win": {
                     "desktop": true,
+                    "app_user_model_id": "Spyder-IDE.Spyder",
```

Comment:
According to my tests with `napari`, Qt seems to be setting this quite strictly. In our case it's `napari.napari.viewer.PKG_VERSION` (with PKG_VERSION replaced appropiately).

You can check which one you get in Spyder once running with this snippet:

```python
import ctypes
from ctypes import wintypes

lpBuffer = wintypes.LPWSTR()
AppUserModelID = ctypes.windll.shell32.GetCurrentProcessExplicitAppUserModelID
AppUserModelID(ctypes.cast(ctypes.byref(lpBuffer), wintypes.LPWSTR))
appid = lpBuffer.value
ctypes.windll.kernel32.LocalFree(lpBuffer)
print(appid)
```

Reply:
Thanks! I'll replace with the appropriate value.